Gordon360/ApiControllers/MembershipsController.cs

```diff
@@ -541,5 +541,18 @@ public IHttpActionResult Delete(int id)
 
             return Ok(result);
         }
+
```

Comment:
This route does expose whether or not an ID exists in our system, since an invalid id will get an "Account not found" response. However, since you have to be authenticated to call the route and the data exposed is minimal, it seems fine to me to leave this in for testing purposes.